quickstart/README.md

```diff
@@ -0,0 +1,54 @@
+# Quarkus demo: Cassandra Client
+
+This example showcases how to use the Cassandra client with Quarkus. 
+
+# Cassandra instance - Running with Docker
+
+Just run it as follows:
+`docker run \
+   --name local-cassandra-instance \
+   -p 7000:7000 \
+   -p 7001:7001 \
+   -p 7199:7199 \
+   -p 9042:9042 \
+   -p 9160:9160 \
+   -p 9404:9404 \
+   -d \
+   launcher.gcr.io/google/cassandra3`
```

Comment:
I suppose another way to cut down on these ports would be to expose a k8s config that would run C* + quickstart container... that would only need to expose port 8080.  But that's probably beyond the scope of a quickstart app.

Reply:
I think that not all people using starters will be familiar with kubernetes. 
The current setup is easier because it starts the Cassandra on local using docker, and users can interact with it because all ports are exposed.

Reply:
Yeah, I think that's right @tomekl007 ... for a quickstart approach this seems more straightforward.